snapshot-labs/snapshot
# feat: Boosting on Base network

### Summary
Support boosts on base network

### How to test
- Create boost using base tokens
- You should see boost network and tokens while creating boost
- You should see boosts using base network on proposal page
